userprofile/templates/userprofile/profile.html

```diff
@@ -2,96 +2,120 @@
+	<div class="col-sm-offset-2 col-sm-8 col-lg-4">
+		<h2>{% trans "A+ Preferences:" %}</h2>
+		<label for="api-access-token">{% trans "API Access Token" %}</label>
+		<div class="input-group">
+			<input readonly type="text" class="form-control read-only" id="api-access-token"
+				value="{{ profile.api_token }}">
+			<span class="input-group-btn">
+				<button data-toggle="tooltip" data-placement="bottom"
+					title="{% trans 'Copy API Access Token' %}" aria-label="{% trans 'Copy API Access Token' %}"
+					class="aplus-button--secondary aplus-button--md js-copy" type="button"><span
+						class="glyphicon glyphicon-copy"></span><span class="">{% trans 'Copy' %}</span></button>
+			</span>
+		</div>
+		<br>
+		<div>
+			<label for="local-storage-services">{% trans "Automatically log in to:" %}</label>
+			<p id="services-instructions">
+				{% blocktrans trimmed%}
+				These are all services that you have marked for automatic redirection on this device. You can remove
+				automatic redirection here and you will be shown the privacy information next time you are being
+				redirected to the service.
 				{% endblocktrans %}
 			</p>
-			<ul class="local-storage-fields" data-forget-text="{% trans "Forget automatic redirection" %}"></ul>
-			</dd>
+			<ul id="local-storage-services" class="local-storage-fields"
+			data-forget-text="{% trans 'Forget automatic redirection' %}" role="group"></ul>
 		</div>
-    </div>
-    <div class="col-md-4">
-
-        {# TODO: list enrolled courses #}
-    </div>
+		<form action="{% url 'set-user-language' %}" method="post" class="form form-horizontal">{% csrf_token %}
+			<div class="form-group">
+				<label class="col-xs-12" for="language-profile">{% trans 'Language' %}</label>
+				<div class="col-xs-12 col-sm-8">
+					<select name="language" class="form-control" id="language-profile">
+						{% get_current_language as LANGUAGE_CODE %}
+						{% get_available_languages as LANGUAGES %}
+						{% get_language_info_list for LANGUAGES as languages %}
+						{% for language in languages %}
+						<option value="{{ language.code }}" {% if language.code == LANGUAGE_CODE %}selected{% endif %}>
+							{{ language.name_local }}
+						</option>
+						{% endfor %}
+					</select>
+				</div>
+				<div class="col-xs-offset-0 col-xs-6 col-sm-4 ">
+					<button type="submit"
+						class="aplus-button--default aplus-button--md btn-block btn-save-profile">{% trans 'Save' %}</button>
+				</div>
+			</div>
+		</form>
+	</div>
 </div>
+<div class="row">
+	<div class="col-sm-offset-2 col-sm-8 col-md-4">
+		{% if not is_google %}
+			{% brand_university_name as university %}
+				<h2>{% blocktrans with university=university trimmed %}Information provided by {{ university }}{% endblocktrans %}</h2>
+		{% elif is_google %}
+			<h2>{% trans "Information provided by Google:" %}</h2>
+		{% endif %}
+		<p class="text-warning">{% trans "The following settings should be changed in your identity provider" %}.</p>
```

Comment:
The ending period should be part of the translated string since different languages may have different punctuation rules.